bench/suite/src/main/scala/org/scalameta/bench/SymbolParser.scala

```diff
@@ -0,0 +1,29 @@
+package org.scalameta.bench
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations.{Benchmark, BenchmarkMode, Measurement, Mode, Fork}
+import org.langmeta.semanticdb.Symbol
+import org.langmeta.semanticdb.Signature
+
+object SymbolParser {
+  val symbol: String = Symbol.Global(
+    Symbol.Global(
+      Symbol.Global(
+        Symbol.Global(
+          Symbol.None,
+          Signature.Term("_root_")),
+        Signature.Term("scala")),
+      Signature.Term("Predef")),
+    Signature.Type("Int")).syntax
+}
+class SymbolParser {
+
+  @Benchmark
+  @BenchmarkMode(Array(Mode.Throughput))
+  @Fork(value = 1)
+  @Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
+  def run: Symbol = {
+    Symbol(SymbolParser.symbol)
```

Comment:
Here is a random sample of symbols that we can benchmark https://gist.github.com/olafurpg/88b041fc027b45d451114284ba40afa5

Reply:
Oops, I didn't really mean to push the commit with the Benchmark just yet 😆. Oh well, thanks for the representative sample of symbols, I'll make a benchmark based on that.